### DISCUSSION TITLE
Lua source of default config

### DISCUSSION THREAD

**Discussion by ghost** [wezterm/wezterm#1327] [Q&A] [answered]
Is it available?

**Comment by wez** [comment, marked as answer]
There isn't literally a default lua config file: wezterm is written in Rust, and the default configuration is a collection of Rust structs with some logic to compute some of the defaults depending on the system. That configuration definition and logic is spread across a number of Rust files in the [`config`](https://github.com/wez/wezterm/tree/main/config/src) subdirectory of the repo.

The configuration and its behavior are documented and explained [in the docs](https://wezfurlong.org/wezterm/config/files.html)
